src/main/java/org/traccar/protocol/PretraceProtocolDecoder.java

```diff
@@ -37,7 +39,7 @@ public PretraceProtocolDecoder(Protocol protocol) {
             .text("(")
             .number("(d{15})")                   // imei
             .number("Uddd")                      // type
-            .number("d")                         // gps type
+            .number("(d)")                         // gps type
```

Comment:
Comment indentation is off

Reply:
This is needed to use the gps type - see below.